boards/arm/nrf54l15pdk_nrf54l15/doc/index.rst

```diff
@@ -0,0 +1,144 @@
+.. _nrf54l15pdk_nrf54l15:
```

Comment:
90KiB image still seems present which @kartben asked to reduce

Reply:
Changed